### PR TITLE
fix: fix routing after upgrade

### PR DESCRIPTION
**Pull request by janmichek**
## Description
After the infrastructure upgrade and following fixes, I discovered one more regression

## Demo

https://github.com/aeternity/aescan/assets/15363559/2bf0832b-c581-4bf9-a580-68c1087fd7ad

## Checklist:
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] fix routing after upgrade

### CODE REVIEW

File: src/components/OraclesPanel.vue [hunk 1]
```diff
@@ -25,9 +25,6 @@ import { storeToRefs } from 'pinia'
 import { computed, ref } from 'vue'
 import { useRoute, useRouter } from 'nuxt/app'
 import { useOraclesStore } from '@/stores/oracles'
-import OraclesTableCondensed from '@/components/OraclesTableCondensed'
```
Comment (janmichek) [inline src/components/OraclesPanel.vue:28]: Apparently after the upgrade the imports that can be auto-imported are removed by linter. Which is a good thing

**Review by Liubov-crypto — COMMENTED** [review]
It seems that there is some issue with displaying Names in Auction - they are not visible and they become visible if I'm opening console to see issues: 

https://github.com/aeternity/aescan/assets/69896204/0c9882c9-c11f-4892-a4ad-cf2c2f8f2607


https://github.com/aeternity/aescan/assets/69896204/db4fe4b8-2372-4d93-9474-4b2a389af775

@janmichek please check it.

**Comment by janmichek** [comment]
> It seems that there is some issue with displaying Names in Auction - they are not visible and they become visible if I'm opening console to see issues:
> 2023-10-31.5.11.00.mov
> 2023-10-31.5.14.25.mov
> 
> @janmichek please check it.

You are right, fixed

**Review by Liubov-crypto — APPROVED** [review]
Fixed, LGTM

**Review by michele-franchi — CHANGES_REQUESTED** [review]
It breaks the current view when you select again the active value:


https://github.com/aeternity/aescan/assets/11598532/9a444e06-f6c7-4e3c-b8f7-04a0cee149bc



Could you have a look at it?

**Comment by janmichek** [comment]
> It breaks the current view when you select again the active value:
> filters.mov
> 
> Could you have a look at it?

My bad, sorry. Fixed

**Comment by Liubov-crypto** [comment]
> https://pr-561-aescan.stg.aepps.com

Reproducible on Accounts page:

https://github.com/aeternity/aescan/assets/69896204/89bb1add-9e34-41b5-b1ed-ebf41719acd4

**Comment by janmichek** [comment]
> > https://pr-561-aescan.stg.aepps.com
> 
> Reproducible on Accounts page:
> 2023-11-02.4.29.42.mov

Found out it was an issue even before. It was present on main deployment as well. I solved it in a different approach - reused from other selects. The selected option is hidden from select options, so this cannot even happen.

**Comment by Liubov-crypto** [comment]
> https://pr-561-aescan.stg.aepps.com



> > > https://pr-561-aescan.stg.aepps.com
> > 
> > 
> > Reproducible on Accounts page:
> > 2023-11-02.4.29.42.mov
> 
> Found out it was an issue even before. It was present on main deployment as well. I solved it in a different approach - reused from other selects. The selected option is hidden from select options, so this cannot even happen.

Fixed, LGTM.

**Review by michele-franchi — APPROVED** [review]
Looks good to me

**Pull request merged**